README.md

```diff
@@ -135,6 +135,33 @@ You can then link the JavaScript pack in Rails views using the `javascript_pack_
 <%= stylesheet_pack_tag 'application' %>
 ```
 
+The `javascript_pack_tag` and `stylesheet_pack_tag` helpers will include all the transpiled
+packs with the chunks in your view, which creates html tags for all the chunks.
+
+The result looks like this:
+
+```erb
+<%= javascript_pack_tag 'calendar', 'map' %>
+
+<script src="/packs/vendor-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+<script src="/packs/calendar~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+<script src="/packs/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+<script src="/packs/map~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+<script src="/packs/map-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+```
+
+**Important:** Pass all your pack names when using this helper otherwise you will
+get duplicated chunks on the page.
+
+```erb
+<%# DO %>
+<%= javascript_packs_with_chunks_tag 'calendar', 'map' %>
```

Comment:
guess it would be `javascript_pack_tag` rather than `javascript_packs_with_chunks_tag`